datafusion-cli/tests/cli_integration.rs

```diff
@@ -51,6 +51,163 @@ fn init() {
     ["--command", "show datafusion.execution.batch_size", "--format", "json", "-q", "-b", "1"],
     "[{\"name\":\"datafusion.execution.batch_size\",\"value\":\"1\"}]\n"
 )]
+
```

Comment:
Add rich testing cases in cli_integration, it can avoid regression for datafusion-cli changes.